twoliter/src/cmd/build.rs

```diff
@@ -23,12 +37,102 @@ pub(crate) struct BuildVariant {
     project_path: Option<PathBuf>,
 
     /// The architecture to build for.
-    #[clap(long = "arch", default_value = "x86_64")]
+    #[clap(long = "arch", env = "BUILDSYS_ARCH", default_value = "x86_64")]
     arch: String,
+
+    /// The variant to build.
+    #[clap(env = "BUILDSYS_VARIANT")]
```

Comment:
In the long term these environment variables should become internal implementation details of the `Makefile.toml`. We shouldn't propagate them outward into the "new" interface for building Bottlerocket. Users can define whatever environment variables they want, but I think we should favor passing values like these as arguments.

```suggestion
    #[clap(long)]
```

Reply:
I've removed all env variables.